### DISCUSSION TITLE
Linux Bluetooth Sync Problem?

### DISCUSSION THREAD

**Discussion by NyaLinux** [ppy/osu#30417] [Q&A]
[osu.webm](https://github.com/user-attachments/assets/a4d7152c-8e54-4d7a-9a9e-6ac364ec76b0)

Hello, I know this has been a problem for quite a while but it seems no one has uploaded a video of it. Using Bluetooth headphones on Linux, using any codec, results in only certain elements being out of sync. Clicking on the settings and navigating throughout the UI is in sync as shown in video. However, the hovering above the cookie and in gameplay is very out of sync. Adjusting the audio offset does nothing to remedy this. I am not sure if this is an osu! issue or a Linux issue but I just wanted to upload a video of what is sounds and looks like.

Edit: Sorry about the quality I had to get it below 10MB and didn't want to shorten it.

**Comment by peppy** [comment]
Adjusting the audio offset should work.
